plugins/circleci/.toolkitrc.yml

```diff
@@ -1,2 +1,5 @@
 installs:
   CircleCiConfig: './lib/circleci-config'
+
+init:
+  - './lib/env-vars'
```

Comment:
**thought**: maybe the file should include `init` in its name (innit) to make its purpose clearer?